com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -715,7 +715,9 @@ public void Shutdown()
             }
             networkTickSystem = null;
 
+#if !UNITY_2020_2_OR_LATER
```

Comment:
@kvassall-unity — I hope you don't mind me asking but can this PR wait for a few days (1-3 days) until we merge other PRs first? :)